packages/dappmanager/src/modules/wireguard/client.ts

```diff
@@ -57,15 +56,19 @@ export class WireguardClient {
     });
   }
 
+  // Wireguard API
+  // - remote:    '/dappnode_admin'
+  // - remote qr: '/dappnode_admin?qr'
+  // - local:     '/dappnode_admin?local'
+  // - local qr:  '/dappnode_admin?local&qr'
   async getDeviceCredentials(
     device: string
   ): Promise<WireguardDeviceCredentials> {
-    const remoteConfigUrl = urlJoin(
+    const remoteConfigUrl = urlJoin(WIREGUARD_API_URL, device);
+    const localConfigUrl = `${urlJoin(
       WIREGUARD_API_URL,
-      WIREGUARD_REMOTE,
       device
-    );
-    const localConfigUrl = urlJoin(WIREGUARD_API_URL, WIREGUARD_LOCAL, device);
+    )}?${WIREGUARD_LOCAL}`;
```

Comment:
This must not be a param, just use a string directly here.
```
const url = urlJoin(WIREGUARD_API_URL, device);
const remoteConfigUrl = url;
const localConfigUrl = `${url}?local=true`
```

Reply:
done on last commit